content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -112,7 +112,17 @@ that updates those files from a remote source, as in the following diagram:
 
 {{< figure src="/images/docs/pod.svg" alt="Pod creation diagram" class="diagram-medium" >}}
 
```

Comment:
Try this:
```markdown

#### API for sidecar containers

{{< feature-state for_k8s_version="v1.28" state="alpha" >}}

```

and then at the end of the new text, another 4th level heading:

```markdown
#### Resource sharing within containers
```

That makes it clearer where the feature state applies.

Reply:
We usually avoid 4th level headings, but for this I'd OK the use.